initial_pose:
  translation: [0.0, 0.0, 0.0]
  rotation_matrix: [1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0]

extrinsics:
  imu_to_lidar:
    translation: [0.006253, -0.011775, 0.007645]
    rotation_matrix: [1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0] 

imu_params:
  acc_scale: 9.79

keyframe:
  distance_th: 2.0
  angle_th: 1000.0 # degree
  min_active_points_rate: 0.95

scan_cloud_preprocess:
  clip_range: 2.0
  filter_size: 0.25

normal_map:
  num_keyframes: 20
  filter_size: 0.25
  num_normal_points: 5
  normal_eigen_val_thresh: 0.001

estimator:
  use_loose_coupling: false
  huber_delta: 0.5
  num_max_iteration: 10
  num_max_matching_points: 100000
  max_correspondence_dist: 1.0
  convergence_th: 0.5
